user-org-actor/src/main/java/org/sunbird/user/UserReadActor.java

```diff
@@ -23,29 +28,43 @@
 )
 public class UserReadActor extends BaseActor {
 
-  IUserDao userESDao = (IUserDao) UserDaoFactory.getDaoImpl(DaoImplType.ES.getType());
+    private IUserService userService = null;
+    IUserDao userESDao = (IUserDao) UserDaoFactory.getDaoImpl(DaoImplType.ES.getType());
+
+   @Override
+   public void onReceive(Request request) throws Throwable {
+       if (UserActorOperations.READ_USER_BY_ID
+               .getOperation()
+               .equalsIgnoreCase(request.getOperation())) {
+           readUserById(request);
+       } else {
+           onReceiveUnsupportedMessage(this.getClass().getName());
+       }
+   }
 
-  @Override
-  public void onReceive(Request request) throws Throwable {
-    if (UserActorOperations.READ_USER_BY_ID
-        .getOperation()
-        .equalsIgnoreCase(request.getOperation())) {
-      readUserById(request);
-    } else {
-      onReceiveUnsupportedMessage(this.getClass().getName());
-    }
-  }
 
-  /**
-   * this method is used to read user from elastic search.
-   *
-   * @param request
-   * @throws BaseException
-   */
-  public void readUserById(Request request) throws BaseException {
-    startTrace("readUserById");
-    Response response = userESDao.getUserById((String) request.getRequest().get(JsonKey.USER_ID));
-    endTrace("readUserById");
-    sender().tell(response, self());
-  }
+    /**
+     * this method is used to read user from elastic search.
+     *
+     * @param request
+     * @throws BaseException
+     */
+    public void readUserById(Request request) throws BaseException {
+        startTrace("readUserById");
+        Response response = null;
+        try {
+            response = userESDao.getUserById((String) request.getRequest().get(JsonKey.USER_ID));
+        } catch (Exception e) {
+            ProjectLogger.log(
+                    "UserReadActor:readUserById: "
+                            + "Exception in getting the record from ES : "
+                            + e.getMessage(),
+                    LoggerEnum.ERROR.name());
+            ProjectLogger.log("Exception occurred while reading user ES.", e);
```

Comment:
remove  ProjectLogger.log("Exception occurred while reading user ES.", e); because we already have logger above.